### PR TITLE
Add SixikUtils Docs

### PR DESCRIPTION
**Pull request by 8sixik**

### CODE REVIEW

**Review by Witixin1512 — CHANGES_REQUESTED** [review]
Requires a couple of changes. I'd prefer it if there were a lot more comments on methods that are not immediately obvious to anyone who doesn't have the source at hand, as well as better naming of methods and parameters. Some methods have parameters with no name and explanation, especially booleans.

File: docs_exported/1.19/sixikutils/docs/mods/sixikutils/curios/player/Player.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Player
+
+## Importing the class
+
+It might be required for you to import the package if you encounter any issues (like casting an Array), so better be safe than sorry and add the import at the very top of the file.
+```zenscript
+import crafttweaker.api.entity.type.player.Player;
```
[the remark this answers is not in the document]
Comment (8sixik) [reply docs_exported/1.19/sixikutils/docs/mods/sixikutils/curios/player/Player.md:7]: In the title?

File: docs_exported/1.19/sixikutils/docs/mods/sixikutils/ftbquest/events/ChapterStartedEvent.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+:::
+
+:::group{name=getObject}
+
+Return Type: [Chapter](/mods/sixikutils/ftbquest/quests/Chapter)
+
+```zenscript
+// ChapterStartedEvent.getObject() as Chapter
+
+myChapterStartedEvent.getObject();
```
[the remark this answers is not in the document]
Comment (8sixik) [reply docs_exported/1.19/sixikutils/docs/mods/sixikutils/ftbquest/events/ChapterStartedEvent.md:52]: GetObject is the quest ID, and get Chapter gets the Chapter ID directly

File: docs_exported/1.19/sixikutils/docs/mods/sixikutils/curios/player/Player.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Witixin1512) [reply docs_exported/1.19/sixikutils/docs/mods/sixikutils/curios/player/Player.md:7]: Yes, see how CraftTweaker does it for Tags for example.

File: docs_exported/1.19/sixikutils/docs/mods/sixikutils/utils/events/entity/player/PlayerBeforeLoadedChunkEvent.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+
+## Extending PlayerEvent
+
+PlayerBeforeLoadedChunkEvent extends [PlayerEvent](/forge/api/event/entity/player/PlayerEvent). That means all methods available in [PlayerEvent](/forge/api/event/entity/player/PlayerEvent) are also available in PlayerBeforeLoadedChunkEvent
+
+## Methods
+
+:::group{name=getChunk}
+
```
Comment (Witixin1512) [inline docs_exported/1.19/sixikutils/docs/mods/sixikutils/utils/events/entity/player/PlayerBeforeLoadedChunkEvent.md:22]: Same as for the After Event

**Review by Witixin1512 — CHANGES_REQUESTED** [review]
A lot of things have improved, but there's a couple of minor changes required: Like invalid classes or typos.
Also double check the previous review, as there are some things that still aren't done from that.

**Comment by 8sixik** [comment]
I decided I'd better rewrite everything. Because when I made this mod and wiki, I didn't know anything about java and because of this there is a lot of confusion.

**Pull request closed without merging**